web.py

```diff
@@ -40,8 +41,14 @@
 
 def process_email_alerts():
     with app.app_context():
-        #utils.email_notification_alert(logger, subject, msg, recipient)
-        pass
+        data = dasset.account_balances()
+        for balance in data:
+            if balance.symbol == 'NZD':
+                balance_format = dasset.asset_dec_to_str(balance.symbol, balance.available)
+                subject = f"Available {balance.symbol} Balance below the minimum threshold"
+                msg = f"Available {balance.symbol} Balance needs to be replenished in the dasset account.<br/><br/>Available {balance.symbol} balance is: ${balance_format}"
+                if balance.available < app.config["MIN_AVAILABLE_NZD_BALANCE_CENTS"]:
```

Comment:
you are comparing a decimal to an int

Reply:
removed the balance_format since it already returning decimal

Reply:
re-added the balance_format